.gitignore

```diff
@@ -0,0 +1,17 @@
+# Ignore Mac system files
+.DS_store
+
```

Comment:
### Wrong Place for Summary
![summary](https://user-images.githubusercontent.com/93039359/185093371-05ef03ed-b9ac-4955-9792-539235a1a23d.png)

### RIght Place for Summary
![right1](https://user-images.githubusercontent.com/93039359/185093390-e4c2a8fc-0f9f-423c-8792-bd098a511be2.png)

- [ ] You should not add your summary in the message. Please follow the steps to update your summary :blush:
- **Step 1**

![step1](https://user-images.githubusercontent.com/93039359/185094224-afdd945d-0ac8-4c9d-8d6a-f542c7007aba.png)
- **Step 2**

![step2](https://user-images.githubusercontent.com/93039359/185094263-c25668c1-e7a7-47a8-87f6-1d0ebb2ef68e.png)

- **Step 3**
![step3 (copy)](https://user-images.githubusercontent.com/93039359/185094283-8c22e596-1b40-4e06-acd1-901911bda96d.png)